### PR TITLE
[improve][doc] Add options to stats and partitioned-stats

### PR DESCRIPTION
**Pull request by AlvaroStream**
### Motivation

I noticed there were some options not shown in the documentation but offered in the help of the tool.

### Modifications

Add the options to stats and partitioned-stats in topics.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [ ] Dependencies (add or upgrade a dependency)
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)

- [ ] `doc-not-needed` 
(Please explain why)

- [ X] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### Matching PR in forked repository

PR in forked repository:

### CODE REVIEW

**Review by mattisonchao — APPROVED** [review]

File: site2/docs/reference-pulsar-admin.md [hunk 1]
```diff
@@ -2429,6 +2429,15 @@ Usage
 pulsar-admin topics stats topic
 ```
 
+Options
+
+|Flag|Description|Default|
+|---|---|---|
+|`-etb`,`--get-earliest-time-in-backlog` | Set true to get earliest time in backlog | false |
```
Comment (momo-jun) [inline site2/docs/reference-pulsar-admin.md:2436]:
```suggestion
|`-etb`,`--get-earliest-time-in-backlog` | Set to `true` to get the earliest time in backlog | false |
```

File: site2/docs/reference-pulsar-admin.md [hunk 2]
```diff
@@ -2429,6 +2429,15 @@ Usage
...
+|`-gpb`,`--get-precise-backlog`| Set true to get precise backlog | false |
```
Comment (momo-jun) [inline site2/docs/reference-pulsar-admin.md:2437]:
```suggestion
|`-gpb`,`--get-precise-backlog`| Set to `true` to get precise backlog | false |
```

File: site2/docs/reference-pulsar-admin.md [hunk 3]
```diff
@@ -2429,6 +2429,15 @@ Usage
...
+|`-sbs`,` --get-subscription-backlog-size`| Set true to get backlog size for each subscription, locking required | false |
```
Comment (momo-jun) [inline site2/docs/reference-pulsar-admin.md:2438]:
```suggestion
|`-sbs`,` --get-subscription-backlog-size`| Set to `true` to get backlog size for each subscription, locking required | false | 
```

File: site2/docs/reference-pulsar-admin.md [hunk 4]
```diff
@@ -2466,8 +2475,11 @@ Options
 
 |Flag|Description|Default|
 |---|---|---|
+|`-gpb`,`--get-precise-backlog`| Set true to get precise backlog | false |
```
Comment (momo-jun) [inline site2/docs/reference-pulsar-admin.md:2478]:
```suggestion
|`-gpb`,`--get-precise-backlog`| Set to `true` to get precise backlog | false |
```

File: site2/docs/reference-pulsar-admin.md [hunk 5]
```diff
@@ -2466,8 +2475,11 @@ Options
...
+|`-sbs`,` --get-subscription-backlog-size`| Set true to get backlog size for each subscription, locking required | false |
```
Comment (momo-jun) [inline site2/docs/reference-pulsar-admin.md:2479]:
```suggestion
|`-sbs`,` --get-subscription-backlog-size`| Set to `true` to get backlog size for each subscription, locking required | false | 
```

**Review by momo-jun — COMMENTED** [review]
@AlvaroStream Thanks for adding the docs! I left a few minor suggestions.
BTW, when looking into [this reference doc](https://pulsar.apache.org/docs/next/reference-pulsar-admin/#remove-message-ttl-1), I noticed the table is in an invalid format. Can you pls add a line break after `Options` to fix it as well? Thank you.

**Review by RobertIndie — APPROVED** [review]

**Review by momo-jun — APPROVED** [review]
LGTM. Thanks for the updates.

**Pull request merged**